app/models/taxon.rb

```diff
@@ -955,7 +981,7 @@ def complete_taxon
   def graftable_if_complete
     return true unless ancestry_changed?
     ct = complete_taxon
-    if ct && ( current_user.blank? || !ct.taxon_curators.where( user: current_user ).exists? )
+    if !@skip_complete && ct && ( current_user.blank? || !ct.taxon_curators.where( user: current_user ).exists? )
```

Comment:
Are you 100% sure this is working? @skip_complete is not the same thing as the `attr_accessor :skip_complete`